msprime/trees.py

```diff
@@ -327,7 +323,10 @@ def draw(self):
         for u in self._tree.nodes():
             v = self._tree.get_parent(u)
             x = self._x_coords[u], self._y_coords[u]
-            dwg.add(dwg.circle(center=x, r=3))
+            if u in self._node_colours and self._node_colours[u] is not None:
```

Comment:
No, I fundamentally disagree with this. If a user makes a mistake, then the library should NOT try to infer what they meant by this mistake and should not try to accommodate it. Libraries should fail early and fail noisily so that the user can fix their code. Simplicity of interface is absolutely key to simple, intuitive and reliable software.

*Please* change this.

There are still no tests, and CI is still failing.